src/lib.rs

```diff
@@ -48,7 +148,6 @@ impl Troffable for Roff {
     }
 }
 
-#[derive(PartialEq, Eq)]
```

Comment:
How come you've been dropping `PartialEq, Eq` from these items?

Reply:
I don't really know why it's derived, I can't imagine it was used for something besides testing (the first full commit has them derived but not used) because comparing manpages for equality sounds very niche.

Reply:
I lean towards keeping them (slightly less important than `Clone` and `Debug`) though we should make sure we call this out as a breaking change.